lib/espresso-runner.js

```diff
@@ -9,13 +10,38 @@ import B from 'bluebird';
 import _ from 'lodash';
 
 
-const TEST_APK_PATH = path.resolve(__dirname, '..', '..', 'espresso-server', 'app', 'build', 'outputs', 'apk', 'androidTest', 'debug', 'app-debug-androidTest.apk');
+const TEST_SERVER_PATH = path.resolve(__dirname, '..', '..', 'espresso-server');
 const TEST_MANIFEST_PATH = path.resolve(__dirname, '..', '..', 'espresso-server', 'AndroidManifest-test.xml');
 const TEST_APK_PKG = 'io.appium.espressoserver.test';
-const REQUIRED_PARAMS = ['adb', 'tmpDir', 'host', 'systemPort', 'devicePort', 'appPackage', 'forceEspressoRebuild'];
+const REQUIRED_PARAMS = ['adb', 'tmpDir', 'host', 'systemPort', 'devicePort', 'appPackage', 'forceEspressoRebuild', 'espressoBuildConfig'];
 const ESPRESSO_SERVER_LAUNCH_TIMEOUT = 30000;
 const TARGET_PACKAGE_CONTAINER = '/data/local/tmp/espresso.apppackage';
 
+/**
+ * Recursively copy all files
+ * @param sourceBaseDir {string} directory to copy files from
+ * @param targetBaseDir {string} directory to copy files to
+ */
+async function copyRecursively (sourceBaseDir, targetBaseDir) {
+  logger.debug(`copyRecursively(${sourceBaseDir}, ${targetBaseDir})`);
```

Comment:
I'd prefer human-readable log string ;)

Reply:
Actually I forgot to remove this log line. Do you think it makes sense to leave it and make more readable?

Reply:
yes, everything that helps to track the flow is helpful